sections/futures/ProfitCalculator/ProfitDetails.tsx

```diff
@@ -0,0 +1,98 @@
+import { BigNumber } from '@ethersproject/bignumber';
+import styled from 'styled-components';
+
+import { PositionSide } from '../types';
+
+export const ProfitDetails = (props: {
+	scalar: number;
+	stopLoss: BigNumber;
+	marketAsset: string;
+	exitPrice: BigNumber;
+	leverageSide: PositionSide;
+	marketAssetPositionSize: BigNumber;
+}) => {
+	const entryOrderDetails = props.leverageSide === PositionSide.LONG ? 'Long' : 'Short';
+
+	return (
+		<>
+			<StyledProfitDetails>
+				{/* ENTRY ORDER */}
+				<RowName>
+					<RowText className="row-name">{'Entry Order:'}</RowText>
+				</RowName>
+				<Details style={{ justifySelf: 'right' }}>
+					<RowText className={props.leverageSide}>{`${entryOrderDetails}`}</RowText>
+					<RowText style={{ marginLeft: '2px' }}>{`,`}</RowText>
+					<RowText style={{ marginLeft: '10px' }}>{`Market`}</RowText>
+				</Details>
+				{/* TAKE PROFIT */}
+				<RowName>
+					<RowText className="row-name">{'Take Profit:'}</RowText>
+				</RowName>
+				<Details style={{ justifySelf: 'right' }}>
+					<RowText>{`Sell`}</RowText>
+					<RowText style={{ marginRight: '10px', marginLeft: '10px' }} className="gray-font-color">
+						{`at`}
+					</RowText>
+					<RowText>{(props.exitPrice.toNumber() / props.scalar).toFixed(2)}</RowText>
+				</Details>
+				{/* STOP LOSS */}
+				<RowName>
+					<RowText className="row-name">{'Stop Loss:'}</RowText>
+				</RowName>
+				<Details style={{ justifySelf: 'right' }}>
+					<RowText>{`Sell`}</RowText>
+					<RowText style={{ marginRight: '10px', marginLeft: '10px' }} className="gray-font-color">
+						{`at`}
+					</RowText>
+					<RowText>{(props.stopLoss.toNumber() / props.scalar).toFixed(2)}</RowText>
+				</Details>
+				{/* SIZE */}
+				<RowName>
+					<RowText className="row-name">{'Size:'}</RowText>
+				</RowName>
+				<Details style={{ justifySelf: 'right' }}>
+					<RowText style={{ marginRight: '10px' }}>
+						{(props.marketAssetPositionSize.toNumber() / props.scalar).toFixed(2)}
+					</RowText>
+					<RowText className="gray-font-color">{`${props.marketAsset}-PERP`}</RowText>
```

Comment:
This displays `sETH-PERP` where other parts of the app display `ETH-PERP`. You can find the code to format this elsewhere.